Int overflow on 32 bits arches

Golang 1.12.6 on i686 and armv7:

```
Testing    in: /builddir/build/BUILD/nuts-0.3.0/_build/src
         PATH: /builddir/build/BUILD/nuts-0.3.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/nuts-0.3.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/jmank88/nuts/version=0.3.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/jmank88/nuts
github.com/jmank88/nuts
FAIL	github.com/jmank88/nuts [build failed]
BUILDSTDERR: # github.com/jmank88/nuts [github.com/jmank88/nuts.test]
BUILDSTDERR: ./key_test.go:59:11: constant 2147483648 overflows int
BUILDSTDERR: ./key_test.go:68:11: constant 549755813888 overflows int
BUILDSTDERR: ./key_test.go:77:11: constant 140737488355328 overflows int
BUILDSTDERR: ./key_test.go:86:11: constant 36028797018963968 overflows int
BUILDSTDERR: ./key_test.go:95:11: constant 1152921504606846976 overflows int
```

--- a/key_test.go
+++ b/key_test.go
@@ -30,7 +30,7 @@ func TestKeyLen(t *testing.T) {
 
 func TestKey(t *testing.T) {
 	for _, test := range []struct {
-		max int
+		max int64
 		xs  []uint64
 		bs  [][]byte
 	}{
@@ -101,7 +101,7 @@ func TestKey(t *testing.T) {
 			},
 		},
 	} {
-		t.Run(strconv.Itoa(test.max), func(t *testing.T) {
+		t.Run(strconv.FormatInt(test.max, 10), func(t *testing.T) {
 			k := make(Key, KeyLen(uint64(test.max)))
 			for i, x := range test.xs {
 				k.Put(x)

